### PR TITLE
Database Search backend

### PR DESCRIPTION
**Pull request by JosephCottingham**

### CODE REVIEW

**Comment by JosephCottingham** [comment]
Make sure to Squash and merge!

File: music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/models.py [hunk 1]
```diff
@@ -114,6 +134,17 @@ class Meta:
         db_table = 'song'
 
     def to_dict(self):
+        print(f'Create {self.title} Dict Format')
```
Comment (tristann3) [inline music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/models.py:137]: We should remove all print statements on merge.

**Review by tristann3 — CHANGES_REQUESTED** [review]
Not a whole lot going on here that I've ever seen before, just the single print to be removed.

File: music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/models.py (hunk 1 above)
Comment (JosephCottingham) [reply music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/models.py]: Removed

**Pull request merged**